### DISCUSSION TITLE
Problems downloading purchased content in versions 6.2.4 and 6.2.5

### DISCUSSION THREAD

**Discussion by CarambaMamba** [UltimaHoarder/UltimaScraper#779] [General]
Hello.
First of all, I want to thank the developers for such a wonderful program. Thank you very much. I just want to apologize for my English.
In the last two versions, I observe the following problem - there is no way to download purchased content. In version 6.2.1 this was, and it was automatic. As soon as I chose to download any content from my list, the purchased photos and videos were placed side by side in separate folders. Readme.dm was unable to solve my problem. Please tell me what I'm doing wrong and how to fix the situation with downloading the purchased content. Thanks!

**Comment by UltimaHoarder** [comment]
You want to separate free and paid content?

**Reply by CarambaMamba** [reply]
Probably we are talking about different concepts. Paid content and purchased content are different. Previously, it was possible to download both automatically and free content. Now only free and paid. But the purchased one, I did not understand how to download.

**Reply by jp1151419** [reply]
> 
> 
> Probably we are talking about different concepts. Paid content and purchased content are different. Previously, it was possible to download both automatically and free content. Now only free and paid. But the purchased one, I did not understand how to download.


I believe that her paid=purchased. There is no distinction done by the program between subscriptions that are free and those that have a monthly fee. The distinction is rather on the posts/messages themselves.

**Reply by CarambaMamba** [reply]
Let me give you an example. I have 7 subscriptions. They have both paid content and free content. And when I enter my personal account, I have the "Purchased" tab. There is also video and photo material for the same 7 subscriptions. I run the script, select scrap everything and after that I do not see in the downloaded what was in the "Purchased" tab. Therefore, I ask how to make the script download what is in the "Purchased" tab? Before version 6.2.4, the script automatically downloaded all the material from there.

**Reply by jp1151419** [reply]
hmm that is strange. Personally I haven't been able to run the script at all for multiple days. I keep getting "AttributeError: " errors which stops the scrape instead of skipping the troublesome account
